src/components/pick-list-item/pick-list-item.tsx

```diff
@@ -202,7 +202,7 @@ export class PickListItem implements ConditionalSlotComponent, InteractiveCompon
   // --------------------------------------------------------------------------
 
   /**
-   * Used to toggle the selection state. By default this won't trigger an event.
+   * Toggle's the selection state. By default this won't trigger an event.
```

Comment:
Should this be "Toggles" instead of "Toggle's"?

Reply:
Good 👁️👁️, thanks for spotting that one!